Podfile.lock

```diff
@@ -73,4 +73,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 6ee5bcd18f0d60a4b82be64479784270153e3f26
 
-COCOAPODS: 1.1.0
+COCOAPODS: 1.0.1
```

Comment:
OK, so updated cocoapods and am now on 1.1.1, so everything is different again! Jeez.